source/d3d11/d3d11_runtime.cpp

```diff
@@ -985,44 +1116,117 @@ namespace reshade::d3d11
 		}
 	}
 
-	void d3d11_runtime::draw_debug_menu()
+	void d3d11_runtime::draw_select_depth_buffer_menu()
 	{
 		if (ImGui::CollapsingHeader("Buffer Detection", ImGuiTreeNodeFlags_DefaultOpen))
 		{
 			bool modified = false;
 			modified |= ImGui::Checkbox("Copy depth before clearing", &_depth_buffer_before_clear);
-			modified |= ImGui::Combo("Depth Texture Format", &_depth_buffer_texture_format, "All\0D16\0D32F\0D24S8\0D32FS8\0");
```

Comment:
Because the destination texture adapt itself to the format of the depth buffer texture, and in all the games i have tested, there is no need to add an extra setting to filter depth textures.
I have tried to ease at max the process of finding the good cleared depth texture.